app/roles/openshift-installer/tasks/main.yml

```diff
@@ -27,12 +27,26 @@
     - "{{ openshift_installer_dir }}/install-config.yaml"
     - "{{ openshift_installer_dir }}/install-config.yaml.bkp"
 
-- name: create ignition files
+- name: cleanup previous install artifacts
   shell: >
```

Comment:
You could consider doing this with the file module:

```ansible
- name: my-state
  file:
    state: absent
    path: {item}
  with_items:
    - firstDir
    - secondDir
```

Reply:
:+1: good call